### ISSUE TITLE
Setting default fields for content types

### ISSUE THREAD

**Issue by Nattusia** [Nattusia/custom_tree#2]
Setting default fields for content types, like "body" could always be presented as one line plain text field instead; how should we do this?

**Comment by mmmikko** [comment]
When someone creates a new content type, drupal will automatically create body field that is "text area" type. It would be slightly better to change that to "text line" (and plain text). But this is not really worth much development time, as this only affects people who make new content types (and they can in couple of minutes change the field anyway).

**Comment by Nattusia** [comment]
Let me clarify. It's still textfield but "flat text" no HTML. Correct?

**Comment by mmmikko** [comment]
Yes. There is rarely case where using html is beneficial, so let's make "flat text" the default.

**Comment by Nattusia** [comment]
This can be done with admin settings.
Go admin menu Configuration ->Content authoring->Text format and editors and disable all formats for authenticated and anonymous user. So only administrator will see the body in the full text format.

**Comment by mmmikko** [comment]
Ok, good.
